src/app/personside/visittkort-v2/body/VisittkortBody.tsx

```diff
@@ -12,75 +13,120 @@ import TilrettelagtKommunikasjon from './tilrettelagtkommunikasjon/TilrettelagtK
 import Vergemal from './vergemal/Vergemal';
 import Sikkerhetstiltak from './sikkerhetstiltak/Sikkerhetstiltak';
 import LenkeBrukerprofil from './lenkebrukerprofil/LenkeBrukerprofil';
-import { useEffect, useState } from 'react';
 import NavKontor from './navkontor/NavKontor';
+import { harFeilendeSystemer } from '../harFeilendeSystemer';
 
 interface Props {
-    person: Person;
+    persondata: PersonData;
 }
 
-function SingleColumnLayout(person: Person) {
+function SingleColumnLayout(persondata: PersonData) {
     return (
         <Kolonne>
-            <Kontaktinformasjon person={person} />
-            <Fullmakter fullmakter={person.fullmakt} />
-            <Familie person={person} />
-            <DeltBosted deltBosted={person.deltBosted} />
-            <Foreldreansvar foreldreansvar={person.foreldreansvar} />
-            <NavKontor navEnhet={person.navEnhet} />
-            <TilrettelagtKommunikasjon tilrettelagtKommunikasjon={person.tilrettelagtKommunikasjon} />
-            <Vergemal vergemal={person.vergemal} />
-            <Sikkerhetstiltak sikkerhetstiltak={person.sikkerhetstiltak} />
+            <Kontaktinformasjon persondata={persondata} />
+            <Fullmakter
+                feilendeSystem={harFeilendeSystemer(persondata, InformasjonElement.PDL_TREDJEPARTSPERSONER)}
```

Comment:
Her får vi tre steder i koden som vi må holde i sync med hverandre. F.eks om `Fullmakter` også ett annet informasjonselement. Lurer derfor på om vi heller burde sende inn `persondata.feilendesystemer`, så kan `Fullmakter` komponenten selv finne ut om det er grunnlag for å vise feilmeldingen. 🤔